launchers/junit/src/testFixtures/java/org/eclipse/dataspaceconnector/junit/launcher/EdcRuntimeExtension.java

```diff
@@ -172,8 +173,7 @@ public void afterTestExecution(ExtensionContext context) throws Exception {
 
     @Override
     protected @NotNull Monitor createMonitor() {
-        return new Monitor() {
-        };
+        return new ConsoleMonitor(logPrefix, ConsoleMonitor.Level.DEBUG);
```

Comment:
While this is great for local testing, should the EDC log at debug level in the pipeline?

Reply:
You're right, I tweaked a bit and probably the best solution would be to use the console monitor unless gradle is launched with the `-quiet` parameter. I modified the code accordingly. I also noticed that the `EdcRuntimeExtension` was shutting down extensions a second time in the `@AfterAll` methods, but the shutdowns were already triggered by an hook defined in the `boot` method.
(example output: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/runs/5551002718)

Reply:
I really like that solution, as it also removes the need to exchange the monitor locally in order to see the test output. And good catch on the double shutdown, never noticed that.

Reply:
seems like the shutdown logs are printed with another monitor, will take a look

Reply:
@ronjaquensel I can't get why the shutdown logs are printed... however they are printed also currently on the pipeline so... if you are ok with this could you approve? thanks

Reply:
@ronjaquensel I found out that the "duplicated shoutdown" is needed by the `DataPlaneIntegrationTests`, so for the moment I left it as it is :(